### DISCUSSION TITLE
discussions disappeared from the main text

### DISCUSSION THREAD

**Discussion by molotro** [knowledgefutures/pubpub#2845] [Support]
We were editing a document, suddenly all the discussions disappeared from the main text. They are still at the bottom of the page, but they are no more visibile in the text. If we add a new discussion it is visibile in the text, but all the old discussion are not visibile. We don't find any "show/hide discussion" button or similar. Does anyone know what may have happened?

**Comment by gabestein** [comment]
Unfortunately, sometimes this occurs due to changes to the text that make keeping discussions anchored impossible. We can sometimes attempt to re-anchor the discussions if it's vital, but it is not always possible.
